platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -670,7 +670,7 @@ public MapData addDataLayer(final String name, final boolean generateCentroid) {
             return mapData;
         }
         final long pointer = nativeMap.addClientDataSource(name, generateCentroid);
-        if (pointer <= 0) {
+        if (pointer == 0) {
```

Comment:
Not to be pedantic, but wouldn't it clarify intention here if it was just `pointer == NULL`? e.g. `pointer <= NULL` wouldn't have made sense to a casual reader.

Reply:
Yep, Java's `null` is not the same as `NULL` in C/C++ and it can't be compared with a `long`.

Reply:
sad